src/libs/language-python/api-parser/tests/modules/one_dawson_class.py

```diff
@@ -0,0 +1,29 @@
+""" A module with a dawson-valid Python class """
+
+class MyClass(dict):
+    """
+        This class is dawson-valid: Lambda will use it as follows:
+
+        obj = MyClass(event, context)
+        return json.dumps(obj)
+
+        Which will return the following JSON:
+
+        {"text": "Hello World"}
+
+    """
+
+    api = {
+        "path": "/test"
```

Comment:
small change needed here: path should not start with a slash (see https://dawson.sh/docs.html#4-function-configuration)